packages/plugins/opentelemetry/src/plugin-utils.ts

```
@@ -0,0 +1,104 @@
+import type { ExecutionRequest } from '@graphql-tools/utils';
+
+export function withState<
+  P,
+  GraphqlState = object,
+  HttpState = object,
+  SubExecState = object,
+>(plugin: WithState<P, HttpState, GraphqlState, SubExecState>): P {
+  let states: {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'states' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)